xwiki-platform-distribution/xwiki-platform-distribution-debian/pom.xml

```diff
@@ -69,6 +69,47 @@
     </extensions>
     <pluginManagement>
       <plugins>
+        <plugin>
```

Comment:
I think you should put that in `<build>` instead of `<pluginManagement>` since you want to execute it for all children (right ?), it would avoid having to do it in each of those children.